docs/NoSSL/lavalink-without-ssl.md

```diff
@@ -7,6 +7,13 @@ description: Without SSL should work out of the box!
 <div data-ea-style="stickybox" class="dark horizontal" data-ea-publisher="darrennathanaelcom" data-ea-type="image"></div>
 
 ---
+### Hosted by @ [MINMIN](http://mininmiix.com)
```

Comment:
Website unreachable? 
![](https://lore.dpaste.org/g/zkbnRV.png)

Reply:
I changed my link to discord invite.

Reply:
Please update the PR accordingly.

Reply:
Can you check it again 😃

Reply:
https://github.com/minimiix-m/lavalink-list/commit/bcf7c8724ab92ce289a236beb4a6395a7bba4327.diff you haven't updated your [branch](https://github.com/minimiix-m/lavalink-list/commits/patch-1/) to reflect the changes.